app/config.py

```diff
@@ -0,0 +1,16 @@
+from pydantic import BaseSettings
```

Comment:
This file seems to be feature specific and should be placed in a subpackage along with all other related files.

Reply:
I wanted to use the general configuration.
but there still isn't any, so this should be unioned with it later

Reply:
ok